pkg/helm/helm.go

```diff
@@ -160,6 +170,7 @@ func (r *Impl) Upgrade(chart *chart.Chart, kubeNamespace string, releaseName str
 	}
 	upgrade := action.NewUpgrade(cfg)
 	upgrade.Namespace = kubeNamespace
+	upgrade.Wait = true
 	return upgrade.Run(releaseName, chart, vals)
```

Comment:
same here add wait to API

Reply:
https://github.com/IBM/the-mesh-for-data/pull/53